esp/scm/espscm.cmake

```diff
@@ -23,6 +23,35 @@
 set ( ESPSCM_SOURCE_DIR ${HPCC_SOURCE_DIR}/esp/scm )
 set ( ESPSCM_GENERATED_DIR ${CMAKE_BINARY_DIR}/generated )
 
+#ifdef _CONTAINERIZED
```

Comment:
This is not how conditions are added to cmake files - the #s will be treated as comments.

Reply:
Sorry, my mistake